packages/google_maps_flutter/google_maps_flutter/test/fake_maps_controllers.dart

```diff
@@ -329,6 +365,49 @@ class FakePlatformGoogleMap {
     return result;
   }
 
+  Set<TileOverlayId> _deserializeTileOverlayIds(List<String> tileOverlayIds) {
+    if (tileOverlayIds == null || tileOverlayIds.isEmpty) {
+      // TODO(iskakaushik): Remove this when collection literals makes it to stable.
+      // https://github.com/flutter/flutter/issues/28312
+      // ignore: prefer_collection_literals
+      return Set<TileOverlayId>();
```

Comment:
I believe collection literals for sets has landed. I think you can replace the line and others below:

```suggestion
      return <TileOverlayId>{};
```

Reply:
This min dart sdk in this plugin is 2.1 so we can't use it yet. Let's keep this as is in this plugin and have a separate PR to remove everything to use collection literal?

Reply:
It can probably be changed in the null safety migration or right after.